test/scripts/fixtures/code_block_with_directives.rst

```diff
@@ -21,7 +21,7 @@ A normal block with parameters.
 
 Text.
 
-::
+.. code-block:: solidity
```

Comment:
This should actually stay as is because it tests behavior with `::` blocks. Test expectations should be updated instead.